### PR TITLE
Updated assets for Ulster unionist party from main vfp site

### PR DESCRIPTION
**Pull request by ryanhyslop**
Copied over the new assets from VfP not the nicest way of doing this but the quickest

### CODE REVIEW

File: src/layouts/default.html.eco [hunk 1]
```diff
@@ -16,11 +16,11 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
         <!-- prod:oldie.css -->
-        <!--[if lte IE 8]><link rel="stylesheet" href="<%- @asset('/dist/css/oldie.css/oldie.css') %>"><![endif]-->
+        <!--[if lte IE 8]><link rel="stylesheet" href="<%- @asset('/dist/css/oldie.css/oldie.css?v=2') %>"><![endif]-->
```
Comment (simonplend) [inline src/layouts/default.html.eco]: This query string isn't needed for cache busting as the DocPad asset plugin creates versioned filenames when the site is built for production.

File: src/layouts/default.html.eco [hunk 2]
```diff
@@ -16,11 +16,11 @@
...
         <!-- endbuild -->
 
         <!-- prod:frontend.css -->
-        <!--[if gt IE 8]><!--><link rel="stylesheet" href="<%- @asset('/dist/css/frontend.css/frontend.css') %>"><!--<![endif]-->
+        <!--[if gt IE 8]><!--><link rel="stylesheet" href="<%- @asset('/dist/css/frontend.css/frontend.css?v=2') %>"><!--<![endif]-->
```
Comment (simonplend) [inline src/layouts/default.html.eco]: Same as above.

**Comment by ryanhyslop** [comment]
Mah bad, have updated the PR

**Comment by simonplend** [comment]
Looks good, merging!

**Pull request merged**